### PR TITLE
Synchronization

### PR DESCRIPTION
**Pull request by falapen**
All the things for using the page for live-viewing twitter.

### CODE REVIEW

File: backend/news-service/src/Routes.js [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 const db = require('./controllers/DatabaseInterface');
 const { addRequest, checkRequestsCompletion } = require('./Requests');
 
 const router = express.Router();
 
+const {
+  MIDDLEWARE_HOST, MIDDLEWARE_PORT,
+} = process.env;
+
+const MIDDLEARE_LIVE_URL = `http://${MIDDLEWARE_HOST}:${MIDDLEWARE_PORT}/live_news`;
```
Comment (antoncarlsson) [inline backend/news-service/src/Routes.js]: Misspelled MIDDLEWARE

File: backend/middleware/src/MicroserviceHandler.js [hunk 2]
```diff
@@ -47,8 +47,12 @@ module.exports = class MicroserviceHandler {
 
       res.status(200).send('Successfully completed request ');
     });
-  }
 
+    this.app.post('/live_news', (req, res) => {
+      const { service, news, timespan } = req.body;
```
Comment (fsandelin) [inline backend/middleware/src/MicroserviceHandler.js]: timespan never used:

```suggestion
      const { service, news } = req.body;
```

File: backend/middleware/src/MicroserviceHandler.js [hunk 3]
```diff
@@ -47,8 +47,12 @@ module.exports = class MicroserviceHandler {
 
       res.status(200).send('Successfully completed request ');
     });
-  }
 
+    this.app.post('/live_news', (req, res) => {
```
Comment (fsandelin) [inline backend/middleware/src/MicroserviceHandler.js]: res never used:

```suggestion
    this.app.post('/live_news', (req) => {
```

Comment (falapen) [reply backend/middleware/src/MicroserviceHandler.js]: Added response 200 code

**Pull request merged**